db_assessment/opConfig/transformers.json

```diff
@@ -607,8 +607,7 @@
                                                         "usrsegatt": [["pkey","STRING"],["con_id","STRING"],["owner","STRING"],["segmen_name","STRING"],["segment_type","STRING"],["tablespace_name","STRING"]],
                                                         "dbahistsysstat": [["pkey","STRING"],["dbid","STRING"],["instance_number","STRING"],["hour","STRING"],["stat_name","STRING"],["cnt","STRING"],["avg_value","STRING"],["mode_value","STRING"],["median_value","STRING"],["min_value","STRING"],["max_value","STRING"],["sum_value","STRING"],["perc50","STRING"],["perc75","STRING"],["perc90","STRING"],["perc95","STRING"],["perc100","STRING"]],
                                                         "dbahistsystimemodel": [["pkey","STRING"],["dbid","STRING"],["instance_number","STRING"],["hour","STRING"],["stat_name","STRING"],["cnt","STRING"],["avg_value","STRING"],["mode_value","STRING"],["median_value","STRING"],["min_value","STRING"],["max_value","STRING"],["sum_value","STRING"],["perc50","STRING"],["perc75","STRING"],["perc90","STRING"],["perc95","STRING"],["perc100","STRING"]],
-                                                        "awrsnapdetails": [["pkey","STRING"],["dbid","STRING"],["instance_number","STRING"],["hour","STRING"],["min_snapid","STRING"],["max_snapid","STRING"],["min_begin_interval_time","STRING"],["max_begin_interval_time","STRING"],["cnt","STRING"],["sum_snaps_diff_secs","STRING"],["avg_snaps_diff_secs","STRING"],["median_snaps_diff_secs","STRING"],["mode_snaps_diff_secs","STRING"],["min_snaps_diff_secs","STRING"],["max_snaps_diff_secs","STRING"]],
-                                                        "opkeylog":[["pkey","STRING"],["opscriptversion","STRING"],["dbversion","STRING"],["hostname","STRING"],["dbname","STRING"],["instance_number","STRING"],["collection_time","STRING"]]
```

Comment:
Hi Divya,

I noticed that "opkeylog" entry is removed from transformers.json. Was that intentional? I wonder if it shouldn't be there with the added new "comments" column in the end (["comments","STRING"]). Like this:

"opkeylog":[["pkey","STRING"],["opscriptversion","STRING"],["dbversion","STRING"],["hostname","STRING"],["dbname","STRING"],["instance_number","STRING"],["collection_time","STRING"],["comments","STRING"]]


Please let me know your thoughts. All other files your coded are reviewed and LGTM.

Reply:
Hi @ersantosnet  There were two rows already in transformers json and one with 7 fields and other with 9 fields. I removed the one with 7 as we get 8 fields from DB export already and we will add one more.  When there were two rows it picked the later entry with 7 fields and it was failing. So removed it.